abdiff/core/init_run.py

```diff
@@ -0,0 +1,37 @@
+"""abdiff.core.init_run"""
+
+import datetime
+import logging
+import os
+from pathlib import Path
+
+from abdiff.config import Config
+from abdiff.core.utils import read_job_json, update_or_create_run_json
+
+CONFIG = Config()
+
+logger = logging.getLogger(__name__)
+
+
+def init_run(
+    job_directory: str,
+    message: str | None = None,
+) -> str:
+    """Function to initialize a new Run as part of a parent Job."""
+    run_timestamp = datetime.datetime.now(tz=datetime.UTC).strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
I _did_ go with UTC time here.  Maybe a little confusing why runs appear are +4-5 hours when viewing locally... but zero ambiguity if we ever decide to leverage S3 which -- I think -- defaults to UTC time for S3 timestamps.